.mailmap

```diff
@@ -498,3 +498,4 @@ Zhi Zhang <willzzhang@tencent.com>
 Zhi Zhang <willzzhang@tencent.com> <zhangz.david@outlook.com>
 Zhi Zhang <willzzhang@tencent.com> <zhangzhi@localhost.localdomain>
 Zhuang Xiaochun <zhuangxc89@163.com>
+Zhang Lei <zhanglei@trendytech.com.cn> <243290414@qq.com>
```

Comment:
@ztczll See: Your entry should be the first of all `Zhang`'s. At Line 483. 

The bad thing happened was: someone else inserted his entries starting with `Guo` in between Z*. But that can be fixed later.